Content.Server/Storage/EntitySystems/StorageSystem.cs

```diff
@@ -601,8 +603,14 @@ public void OpenStorageUI(EntityUid uid, EntityUid entity, ServerStorageComponen
             if (!Resolve(uid, ref storageComp) || !TryComp(entity, out ActorComponent? player))
                 return;
 
+            // prevent spamming bag open / honkerton honk sound
+            TryComp<UseDelayComponent>(uid, out var useDelay);
```

Comment:
i mean either handle the case where TryComp fails or just use Comp

Reply:
wont resolve do nothing if its null

Reply:
yeah thats why i said using Comp would make more sense since TryComp is more expensive

Reply:
requiring a usedelay would be horrific since so many random things (see: artifacts) would need it added
now i just have it use the result of trycomp